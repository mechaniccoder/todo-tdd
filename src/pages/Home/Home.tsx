import React from "react";
import { Link } from "react-router-dom";

const Home = () => {
  con;
  return (
    <div>
      <div>go to sign up</div>
      <button>submit</button>
      {/* <Link to="/signup">go to sign up</Link> */}
    </div>
  );
};

export default Home;
