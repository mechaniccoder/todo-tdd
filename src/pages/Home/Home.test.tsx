import { render, screen } from "@testing-library/react";
import userEvent from "@testing-library/user-event";
import React from "react";
import { BrowserRouter } from "react-router-dom";
import Home from "./Home";

const renderHome = () => {
  return render(<Home />);
};

describe("Home", () => {
  it("snapshot", () => {
    const { container } = renderHome();
    expect(container).toMatchSnapshot();
  });

  // it("navigates to sign up page", () => {
  //   renderHome();
  //   userEvent.click(screen.getByText(/go to/));
  //   expect(location.pathname).toBe("/signup");
  // });
});
