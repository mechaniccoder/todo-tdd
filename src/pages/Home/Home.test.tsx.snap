// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home snapshot 1`] = `
<div>
  <div>
    <div>
      go to sign up
    </div>
    <button>
      submit
    </button>
  </div>
</div>
`;
